.changeset/hungry-boxes-attack.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/cms-api": patch
+---
+
+DAM: Fix headers
+
+While we fixed a few issues with cache control headers in https://github.com/vivid-planet/comet/pull/2653, there are still a few issues with the way the headers were being handled:
+
+-   All headers are stored while we only need the `content-type` header.
+-   Imgproxy headers are passed through to the client.
+-   `content-type` is stored redundantly for AzureStorageAccounts and S3 Buckets.
```

Comment:
This is the old behavior. IMO we should describe the new behavior in the changeset. For instance

- Only store the `content-type` header
- Prevent imgproxy headers from being passed through to the client
- Remove redundantly stored `content-type` for Azure storage accounts and S3 buckets

Reply:
You are right, 934e4be0cd838eb8fccb419ab9dc814530531853